docs/cpp/destructors-cpp.md

```diff
@@ -124,7 +124,7 @@ int main() {
    delete b2;
 }
 
-Output: A3 dtor
+/* Output: A3 dtor
```

Comment:
The thing to do here is break this out of the code block, and add a section immediately after that starts:
Three ticks: ```output
A3 dtor
...
```
I've seen the pattern you are trying here in some of our docs. That's an old pattern and I remove it when I come across it, so don't want to introduce it afresh in new docs.

Reply:
"\`\`\`Output" with a capital "O" has over 1k matches, but "\`\`\`output" with a lowercase "o" only have 60 something hits. Is any one of the above preferred, or it does not matter?

Reply:
It doesn't matter because we are relying on the tag for the code block type to do the rendering for us.